lib/generators/scenic/view/view_generator.rb

```diff
@@ -8,6 +8,10 @@ class ViewGenerator < Rails::Generators::NamedBase
       source_root File.expand_path("../templates", __FILE__)
 
       def create_view_definition
+        unless views_path.exist?
```

Comment:
should this be its own public method? The public methods in thor tasks are the steps in a process. This could be "create_views_directory" which I think would describe the process better.